experimental/packages/opentelemetry-api-metrics/src/types/Metric.ts

```diff
@@ -33,36 +31,18 @@ export interface MetricOptions {
 
   /**
    * The unit of the Metric values.
-   * @default '1'
+   * @default ''
    */
   unit?: string;
-
-  /** The map of constant labels for the Metric. */
-  constantLabels?: Map<string, string>;
-
-  /**
-   * Indicates the metric is a verbose metric that is disabled by default
-   * @default false
-   */
-  disabled?: boolean;
-
-  /**
-   * Indicates the type of the recorded value.
-   * @default {@link ValueType.DOUBLE}
-   */
-  valueType?: ValueType;
-
-  /**
-   * Boundaries optional for histogram
-   */
-  boundaries?: number[];
-
-  /**
-   * Aggregation Temporality of metric
-   */
-  aggregationTemporality?: AggregationTemporality;
```

Comment:
The enum `AggregationTemporality` should also be removed from the package api-metrics since it is not used in api-metrics anymore.